app/api/drink_routes.py

```diff
@@ -0,0 +1,67 @@
+from flask import Blueprint, request, jsonify
+from app.models import db, Menu, Business, Toppings, Drinks
+from ..forms.drink_form import DrinkForm
+from flask_login import current_user, login_required
+from .auth_routes import validation_errors_to_error_messages
+
+drink_routes = Blueprint("drinks", __name__, url_prefix="/drinks")
+
+# Get all drinks + toppings, needs to be tested
```

Comment:
No get all drinks / topping. 

It should be a get drinks / toppings of a Menu. 
Move to Menu routes.